fluent/reference.md

```diff
@@ -618,14 +618,55 @@ $builder->joinedTableInheritance()->column('type');
 ```
 
```

Comment:
This occurred to me too but I wanted to leave intact in case someone has this exact URL bookmarked. Very small chance I guess :)